python/_nimblephysics/simulation/World.cpp

```diff
@@ -169,6 +169,17 @@ void World(py::module& m)
             return self->getIndex(_index);
           },
           ::py::arg("index"))
+      .def(
+          "getNumBodyNodes",
+          +[](dart::simulation::World* self) -> std::size_t {
+            return self->getNumBodyNodes();
+          })
+      .def(
+          "getBodyNodeIndex",
```

Comment:
I see you didn't create this method, but is it really the case that `getBodyNodeIndex()` doesn't return the index of a BodyNode? I think we should rename this to explain better what it does. Maybe `getBodyNodeByIndex()`?

Reply:
Agreed! Opened #59 to fix this in a separate PR.